app/controllers/account/tenants_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Account::TenantsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @tenants = Tenant.all.page(params[:page]).per(10)
+  end
+
+  def show
+    @tenant = current_tenant
+  end
+
+  def new
+    @tenant = Tenant.new
+  end
+
+  def create
+    @tenant = Tenant.new(tenant_params)
+    if @tenant.save
+      flash[:success] = "Tenant is successfully created"
+      redirect_to account_tenant_path(@tenant)
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @tenant = current_tenant
+  end
+
+  def update
+    @tenant = current_tenant
+    if @tenant.update_attributes(tenant_params)
+      flash[:success] = "Information is updated"
```

Comment:
fyi,
you may add custom flash type and use it within `#redirect_to` as option, e.g.:
```
class TenantsController < ApplicationController
  add_flash_types :success

  def update
    if @tenant.update(tenant_params)
      redirect_to account_tenant_path(@tenant), success: 'Information updated successfully'
    else
      render :edit
    end
  end
end
```

Reply:
Thank you for your suggest. I believe I'll can do it after merging my branch with master